src/site/content/en/blog/economic-times-cwv/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post-old
+title: How The Economic Times passed Core Web Vitals thresholds and achieved an overall 43% better bounce rate
+subhead: |
+  Optimizing Core Web Vitals on The Economic Times website significantly improved the user experience while significantly reducing bounce rate across the the entire website.
+description: |
+  Optimizing Core Web Vitals on The Economic Times website significantly improved the user experience while significantly reducing bounce rate across the the entire website.
+authors:
+  - anshusharma
+  - sumitgugnani
+  - prashantmishra
+date: 2022-12-01
+hero: image/dB6B4Sr8kaaT0KZujRBFC303oFR2/rhX18pRJWKzMdDNFzxr7.PNG
+alt: Google Core Web Vitals + Mail.ru Home Page.
+thumbnail: image/dB6B4Sr8kaaT0KZujRBFC303oFR2/7lLVG6IeMKZYyhLanvXc.PNG
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+With internet speeds improving day by day, users expect websites to respond and behave faster than ever. [The Economic Times](https://m.economictimes.com/) handles over 45 million monthly active users. By optimizing for Core Web Vitals across the domain, on AMP and non-AMP pages, we managed to significantly reduce bounce rates and improve the reading experience.
+
+## Measuring the Impact
+
+We focused on [Largest Contentful Paint (LCP)](/lcp/) and[Cumulative Layout Shift (CLS)](/cls/), as they matter the most when it comes to providing a great reading experience to our users. After implementing various performance fixes as described below, The Economic Times managed to improve Chrome User Experients (CrUX) report metrics significantly within a few months.
+
+**Overall, CLS improved** by 250% from 0.25 to 0.09.
+**Overall, LCP improved** by 80% from 4.5 seconds to 2.5 seconds.
+
+Further, LCP values in the "Poor" range were reduced by 33% from October 2020 to July 2021:
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/1mrpePbYaQeKbIDJKo8b.png", alt="A screenshot of LCP distributions grouped by month, starting from October 2020 and ending in July 2021. The amount of LCP values classified as 'Poor' was reduced from 15.03% to 10.08%.", width="800", height="477" %}
```

Comment:
```suggestion
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/1mrpePbYaQeKbIDJKo8b.png", alt="LCP distributions grouped by month, starting from October 2020 and ending in July 2021. The amount of LCP values classified as 'Poor' was reduced from 15.03% to 10.08%.", width="800", height="477" %}
```

Reply:
Per the a11y best practice to not use "A screenshot of..." or "A picture of..." in alt text